content/graph/MaximumClique.h

```diff
@@ -0,0 +1,62 @@
+/**
+ * Author: chilli, Haobin Ni
+ * Date: 2019-05-10
+ * License: CC0
+ * Source: https://en.wikipedia.org/wiki/MaxCliqueDyn_maximum_clique_algorithm,
+ * https://github.com/FTRobbin/Dreadnought-Standard-Code-Library/blob/11c861e55a73be9c32a799bfe4398e0c62c2da15/improve/MaximumClique.cpp
+ * Description: Finds the maximum clique of a graph. Can be used to find the
+ * maximum independent set by finding the clique of the complement graph.
+ * Time: Runs in about ~1s for n=155 and worst case random graphs (p=.95). Runs
+ * faster for sparse graphs.
+ * Status: fuzz-tested
+ */
+struct Maxclique {
+	double limit=0.025, pk=0;
+	struct Vertex { int i, d; Vertex(int i) : i(i), d(0) {} };
+	typedef vector<Vertex> vv;
+	vector<B> e;
```

Comment:
hm, missing a definition of `B`. Use `/// include-line` if needed for the fuzz-test

Reply:
Perhaps it should just be a `vector<bool>`?

Reply:
bitset is fewer indirections, so I'd stick with that. Hm, and defining it within the struct should avoid the need for  `/// include-line`